clickable.json

```diff
@@ -0,0 +1,158 @@
+{
+    "builder": "custom",
+    "build": "echo Run \\'clickable build-libs --debug\\' to build && false",
+    "restrict_arch": "amd64",
+    "libraries": {
+      "unity8": {
+        "builder": "cmake",
+        "src_dir": "${ROOT}",
+        "test": "make xvfballtests -j${NUM_JOBS}",
```

Comment:
Where does `NUM_JOBS` come from? I guess this will run with unlimited build jobs. But you can use [`NUM_PROCS`](https://clickable-ut.dev/en/latest/clickable-json.html#placeholders-environment-variables) instead.

Reply:
My mistake, I fumbled the name of the variable.